pom.xml

```diff
@@ -826,38 +826,38 @@
         <equinox.osgi.version>3.11.0.v20160603-1336</equinox.osgi.version>
         <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
         <carbon.kernel.version>5.2.0</carbon.kernel.version>
-        <carbon.touchpoint.version>1.1.0</carbon.touchpoint.version>
+        <carbon.touchpoint.version>1.1.1</carbon.touchpoint.version>
         <carbon.kernel.version.range>[5.2.0, 6.0.0)</carbon.kernel.version.range>
-        <carbon.analytics.common.version>6.0.2</carbon.analytics.common.version>
+        <carbon.analytics.common.version>6.0.3</carbon.analytics.common.version>
```

Comment:
Shall we bump this to 6.0.4 which is the latest as of now.?

Reply:
Okay. I will do it.